### ISSUE TITLE
[Feature request] Remove played tracks from playlist

### ISSUE THREAD

**Issue by pierg75** [tramhao/termusic#134]
The playlist doesn't seem to be updated anymore when a track is played, i.e. it's not removed, plus the current playing track is always just the first track. It just seems that nothing is updated anymore.
I think this started with the new termusic-server, but I didn't use it for few days before.

**Comment by pierg75** [comment]
Thanks, but I'm not sure I fully got it. The playlist, once you add some files and then play, it's not updated anymore.
Before when a file was fully played, it was removed from the list in the playlist, now they are always there.

**Comment by tramhao** [comment]
Yes, right now the playlist is working like other music players. The playlist can be modified by user, but the player will not modify it. Before the playlist will change order or work like queue, remove after play. But it seems other music player don't do it and user complain about it also. Which way you prefer?

**Comment by pierg75** [comment]
Understood. TBH I don't have a huge preference, but I liked as it was before, i.e. the playlist being like a queue and the entry removed when playing. Not a big issue per se, but the real problem for me now is that I don't have a clue of which song is playing (as the playlist is not updated and the playing song neither).

**Comment by tramhao** [comment]
I've updated the playlist in master branch, to highlight the currently playing track in playlist. You can try if it's ok.

**Comment by pierg75** [comment]
Thanks, that's much better. Do you think is there a way to update the playlist too (i.e. remove the played songs)?
Maybe these can be made as a knob, so everyone would be able to customise them.
